ui/src/components/AlarmFilters.vue

```diff
@@ -93,6 +111,10 @@ const updateList = (severities: string[]) => {
 				<FeatherIcon :icon="CheckCircle" aria-hidden="true" class="icon ack" />
 				<span>Acknowledge</span>
 			</FeatherButton>
+			<FeatherButton class="acction-btn" @click="() => handleRemoveAlarm()">
```

Comment:
I see a few `acction-btn` classes around but no corresponding css rule for them. Maybe it can be removed?
Also should be able to do just `@click=handleRemoveAlarm`